docs/source/guide/benefits/why_open_source.md

```diff
@@ -0,0 +1,25 @@
+(why_open_source)=
+# Why embrace open source?
```

Comment:
I will also add "The community" to build the best software together. also about inclusivity.

Reply:
Thanks. I think given the time constraints I'll leave it with what we have under `Why embrace open source? > Community support` for the moment